projects/js-packages/publicize-components/src/components/media-section/styles.module.scss

```diff
@@ -48,4 +59,4 @@
 		background: #ddd;
 		color: #1e1e1e;
 	}
```

Comment:
The newline here is being removed, is that intentional?

Reply:
hmm, I dont think so, thanks for noticing!